cmd/acikubectl/cmd/policy.go

```diff
@@ -0,0 +1,87 @@
+package cmd
+
+import (
+	"bytes"
+	kubecontext "context"
+	"fmt"
+	"github.com/spf13/cobra"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"os"
+	"strings"
+)
+
+func processpolicy(cmd *cobra.Command, args []string) {
+	// Extract the arguments
+	req := args[0]
+
+	kubeClient := initClientPrintError()
+	if kubeClient == nil {
+		return
+	}
+	systemNamespace, err := findSystemNamespace(kubeClient)
+	if err != nil {
+		fmt.Fprintln(os.Stderr, "Could not find aci-containers system namespace:", err)
+		return
+	}
+
+	systemNamespacePods, err1 := kubeClient.CoreV1().Pods(systemNamespace).List(kubecontext.TODO(), metav1.ListOptions{})
+	if err1 != nil {
+		fmt.Fprintln(os.Stderr, "Could not list pods:", err1)
+	}
+
+	// Check again in kube-system
+	if len(systemNamespacePods.Items) == 0 {
+		systemNamespace = "kube-system"
+		systemNamespacePods, err1 = kubeClient.CoreV1().Pods(systemNamespace).List(kubecontext.TODO(), metav1.ListOptions{})
+	}
+	if err1 != nil {
+		fmt.Fprintln(os.Stderr, "Could not list pods:", err1)
+	}
+
+	if req == "store" {
+		for pod := range systemNamespacePods.Items {
+			if strings.Contains(systemNamespacePods.Items[pod].Name, "aci-containers-host") {
+				buffer := new(bytes.Buffer)
+				path := "/usr/local/var/lib/opflex-agent-ovs/startup/pol.json"
+				execCmd := fmt.Sprintf("gbp_inspect -fprq DmtreeRoot -t dump > %s", path)
```

Comment:
Could you add the -x option to exclude observables ?

Reply:
okay will do